spec/system/order_spec.rb

```diff
@@ -25,5 +27,22 @@
     within('[data-test-id="Kudo_Points"]') do
       expect(page).to have_content '0'
     end
+
+    click_link 'Orders'
+    expect(page).to have_content reward.title
+    expect(page).to have_content reward.description
+    expect(page).to have_content reward.price
+
+    sign_in admin_user
```

Comment:
You could simplify it by using `using_session`

<img width="820" alt="Screenshot 2022-03-07 at 10 14 03" src="https://user-images.githubusercontent.com/5034850/157001839-7d65eb87-f2c1-4503-99ce-18b319414329.png">

Reply:
I added `using_session`